go.mod

```diff
@@ -9,10 +9,79 @@ go 1.13
 replace github.com/spf13/cobra => github.com/spf13/cobra v1.7.0
 
 require (
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.252.1
-	github.com/ONSdigital/dp-rchttp v1.0.0
-	github.com/ONSdigital/go-ns v0.0.0-20200205115900-a11716f93bad // indirect
-	github.com/ONSdigital/log.go/v2 v2.4.1
-	github.com/gorilla/mux v1.8.0
-	github.com/smartystreets/goconvey v1.8.0
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.260.0
+	github.com/ONSdigital/log.go/v2 v2.4.3
+	github.com/gorilla/mux v1.8.1
+	github.com/smartystreets/goconvey v1.8.1
+)
+
+require (
+	github.com/ONSdigital/dp-api-clients-go v1.43.0 // indirect
+	github.com/ONSdigital/dp-frontend-models v1.1.0 // indirect
+	github.com/ONSdigital/dp-healthcheck v1.6.1 // indirect
+	github.com/ONSdigital/dp-mocking v0.10.1 // indirect
+	github.com/ONSdigital/dp-net v1.5.0 // indirect
```

Comment:
that's a pain, what's bringing that in?

Reply:
Commit `Remove github.com/ONSdigital/dp-rchttp` brings in:

- `github.com/ONSdigital/dp-frontend-models v1.1.0 // indirect`

Commit `Add github.com/ONSdigital/dp-api-clients-go/v2/headers package` brings in:

- `github.com/ONSdigital/dp-healthcheck v1.6.1 // indirect`
- `github.com/ONSdigital/dp-mocking v0.10.1 // indirect`
- `github.com/ONSdigital/dp-net v1.5.0 // indirect`